internal/pkg/service/appsproxy/proxy/server.go

```diff
@@ -58,7 +58,7 @@ func StartServer(ctx context.Context, d dependencies.ServiceScope) error {
 
 func NewHandler(d dependencies.ServiceScope) http.Handler {
 	// Setup OAuth2Proxy singleton global logger
-	loggerWriter := logging.NewLoggerWriter(d.Logger(), "info")
+	loggerWriter := logging.NewLoggerWriter(d.Logger().WithComponent("oauth2proxy"), "info")
 	oautproxylogger.SetOutput(loggerWriter)
```

Comment:
Mark logs from  the OAuth2Proxy lib. 
![image](https://github.com/keboola/keboola-as-code/assets/19371734/fd015cc4-20c3-45b6-8eaa-aed1dee10d58)